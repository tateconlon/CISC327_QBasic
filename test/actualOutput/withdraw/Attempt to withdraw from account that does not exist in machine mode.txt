login as machine successful
Account 1234567 does not exist
transaction code 1 invalid
Transaction summary written and system logged out successfully