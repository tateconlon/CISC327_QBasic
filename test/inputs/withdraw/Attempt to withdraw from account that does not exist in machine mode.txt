login
machine
withdraw
1234567
1
logout