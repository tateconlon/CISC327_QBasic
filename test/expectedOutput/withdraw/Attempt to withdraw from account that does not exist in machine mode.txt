login as machine successful
Account 1234567 does not exist
Transaction summary written and system logged out successfully